app/models/api/config.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Config
```

Comment:
Would it make sense to wrap this in Config::Options?  This would make it "look like" ::Settings, and would give nested dot access, so you could do `.base.name` for example.